src/util/common/config.js

```diff
@@ -24,6 +24,20 @@ const DEFAULT_ROOT_FIELDS = [
   'release',
 ];
 
+/**
+ * Array of bunyan's different log levels.
+ * https://github.com/trentm/node-bunyan#levels
+ * @type {Array}
+ */
+export const BUNYAN_LOGGER_LEVELS = [
```

Comment:
Its unlikely, but we'll need to keep this in sync with any changes in bunyan. What if we pulled this from bunyan itself:

```js
import { levelFromName } from 'node-bunyan';

...

export const BUNYAN_LOGGER_LEVELS = Object.keys(levelFromName)
```